main.go

```diff
@@ -57,6 +82,25 @@ func Start() {
 		log.Fatalln(fmt.Errorf("cannot write resolv.conf file: %s", writeResolvErr).Error())
 	}
 
+	netConfig := path.Join(handlers.CNIConfDir, handlers.DefaultCNIConfFilename)
+	_, err = os.Stat(netConfig)
+	if os.IsNotExist(err) {
+		log.Printf("generating network config..\n")
```

Comment:
Can we use `Writing network config..\n`?

Reply:
Sure do I keep all messages lowercase?

Reply:
No, only errors as in fmt.Errorf

Reply:
Done!